.travis.yml

```diff
@@ -5,4 +5,5 @@ cache:
 node_js:
   - 6.9
 before_script:
+  - git fetch --unshallow
```

Comment:
We should not need this anymore after 21997dd3e6a1c90a5856f77cf8a7587236961a3f was merged.

Reply:
We may still need it just for tests, the change was done in `generateStatus.js` and tests don't touch that file, so that path remains broken.

Reply:
Ah, maybe it's still needed here then and given that the tests seems to fail now it kinda points to that.